content/2018-05-02-how-pydelhi-conducts-hiring-pitching-sessions.md

```diff
@@ -0,0 +1,25 @@
+Title: How PyDelhi conducts Hiring and Pitching sessions
+Slug: pydelhi-hiring-pitching-sessions
+Date: 2018-05-02
+Category: General
+Author: Shashank Kumar
+Email: abc@shanky.xyz
+Summary: Hiring and Pitching sessions have been a part of PyDelhi events since the beginning. Here's how it is organized.
+
+![Hiring Pitching Sessions]({filename}/images/hiringpitching.jpg)
+
+Hiring and Pitching sessions have been a part of PyDelhi events since the beginning. The session is scheduled close to the end of every event just before the closing notes. This blog is to help out volunteers and participants by putting organized structure to the entire process. Points raised below are the work of volunteers, open to discussion and will be shaped better with time.
+
+## Step 1 - Broadcasting on social media
+
+The initial step, before even the event start, volunteers should be sharing the link to etherpad which will be used to jot down details of folks speaking during the session. This will help folks to be prepared beforehand and can be called by the volunteers during the session. Also, this will be beneficial for the folks who are not able to attend the event. Official PyDelhi channels (Twitter/FB) should be preferred to broadcast the link. Etherpad link which can be used is https://public.etherpad-mozilla.org/p/Oms9Iw8MK4
+
+## Step 2 - Conducting Session
+
+Close to the end of the event or by the schedule for Hiring and Pitching session, volunteers should open the etherpad link, preferably with the projector connected and call out the folks who submitted details one by one. If the person is not present, their details should be called out by volunteer themselves. After when the pre-submitted details are over, one of the volunteers should call folks from the audience to see if some of the would-be interested in sharing their details.
```

Comment:
Maybe consider replacing `call out the folks who submitted details one by one.` by `invite the speakers to make their pitch one by one.`